docs/src/pages/components/progress/CircularIntegration.tsx

```diff
@@ -68,31 +39,52 @@ export default function CircularIntegration() {
   };
 
   return (
-    <div className={classes.root}>
-      <div className={classes.wrapper}>
+    <Box sx={{ display: 'flex', alignItems: 'center' }}>
+      <Box sx={{ m: 1, position: 'relative' }}>
         <Fab
           aria-label="save"
           color="primary"
-          className={buttonClassname}
+          sx={buttonSx}
```

Comment:
I guess it will be simpler once we support all the colors of the theme (`color="success"`), but we can't yet.